### PR TITLE
Update macOS Copy

### PR DESCRIPTION
**Pull request by markavitale**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Update the website copy per recommendations

#### Focus areas to test

Check the macOS getting started and controls pages

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui/pull/13071)

### CODE REVIEW

File: apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/mac/GetStartedOverview.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
-Fluent UI macOS is an [open-source](https://github.com/microsoft/fluentui-apple) native library that provides the Fluent UI experience for the macOS platform. It contains custom controls and customizations for platform controls, all from the official Fluent design language used in Microsoft 365 products.
+Fluent UI Apple is an [open-source](https://github.com/microsoft/fluentui-apple), native library used by Microsoft 365 apps. We offer customizations and custom controls that deliver unique Fluent experiences to customers using macOS devices.
```
Comment (markavitale) [inline apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/mac/GetStartedOverview.md:1]: It wasn't clear to me if we still wanted open-source to be a link from the feedback.

File: apps/fabric-website/src/pages/Overviews/ControlsPage/docs/mac/ControlsOverview.md [hunk 2]
```diff
@@ -1 +1 @@
-Fluent UI's robust, up-to-date macOS controls are built with Swift and are compatible with Objective-C. Look through the control list to see the building blocks that are available using Fluent UI macOS.
+Fluent UI's robust, up-to-date macOS controls are built with Swift and are compatible with Objective-C. Look through the control list to see the building blocks that are available using Fluent UI Apple.
```
Comment (markavitale) [inline apps/fabric-website/src/pages/Overviews/ControlsPage/docs/mac/ControlsOverview.md:1]: Do we want to make similar Fluent UI macOS -> Fluent UI Apple tweaks for the last "request" section? It wasn't mentioned, but might make sense.
image

Comment (ejlayne) [reply apps/fabric-website/src/pages/Overviews/ControlsPage/docs/mac/ControlsOverview.md:1]: Yes, I think this makes sense to do.

File: apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/mac/GetStartedOverview.md (hunk 1 above)
Comment (ejlayne) [reply apps/fabric-website/src/pages/Overviews/GetStartedPage/docs/mac/GetStartedOverview.md:1]: I would keep it as a link for now, but we can follow-up with Jennifer on the ADO item.

**Pull request merged**